File: src/AHF-API/DataLayer/Model/FinishedGame.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model
{
    public class FinishedGame
    {
        public FinishedGame(int gameId)
        {
            GameId = gameId;
        }

        public int Id { get; set; }
        public int GameId { get; set; }
        public string TeamsLeaderboard { get; set; }
        public string Winner { get; set; }
    }
}

File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/Model/FinishedGame.java
package com.ahf.antwerphasfallen.Model;

/**
 * Created by Jorren on 17/01/2019.
 */

public class FinishedGame {
    private int id;
    private int gameId;
    private String teamsLeaderboard;
    private String winner;

    public int getGameId() {
        return gameId;
    }

    public String getTeamsLeaderboard() {
        return teamsLeaderboard;
    }

    public String getWinner() {
        return winner;
    }
}
